### DISCUSSION TITLE
Getting old alerts

### DISCUSSION THREAD

**Discussion by Shom770** [weather-gov/api#537] [Q&A] [answered]
I'm not able to retrieve alerts issued a month or so again via the /alerts endpoint. I was intending to get the polygon of all the snow squall warnings issued on 2/19/22, though the /alerts endpoint returns nothing for that day. How would I go about getting the polygons?

**Comment by netbymatt** [comment, marked as answer]
The API has between 14 and 30 days of previous alerts. Beyond this
timeframe you can use Iowa State's database to get the polygons for
specific events and dates.

https://mesonet.agron.iastate.edu/request/gis/watchwarn.phtml

Some of the data goes back to 1986. And they've done a nice job of
homogenizing it so the data that you get out is consistent even though how
the products have been presented has changed over the years.
